frontend/src/app/modules/home/issues/issues.component.ts

```diff
@@ -43,11 +48,18 @@ export class IssuesComponent extends BaseComponent implements OnInit {
         event.originalEvent.stopPropagation();
     }
 
+    private showPaginator() {
+        if (this.length > 10) {
+            this.paginators = (this.paginators === false);
+        }
+    }
+
     private loadIssues() {
         this.issueService.getIssues()
             .pipe(this.untilThis)
             .subscribe(response => {
                 this.issues = response;
+                this.length = this.issues.length;
             }, errorResponse => {
                 this.toastNotification.error(errorResponse, 'Error', 1500);
             });
```

Comment:
There is a possibility that the request will be executed later than the condition for the paginator will be checked. It is necessary to ensure the correct sequence of method calls.
```suggestion
    private loadIssues() {
        return this.issueService.getIssues()
            .pipe(
                tap(issues => {
                    this.issues = issues;
                    this.numberOfIssues = issues.length;
                }),
                catchError(error => of(this.toastNotification.error(error, 'Error', 1500)))
            )
    }
```

Reply:
Ok

Reply:
Vlad sorry but your version with comments does not work as mine (

Reply:
With your code, paging is loaded for 5 seconds. This is a bad decision.

Reply:
I tested the code with loading issues and everything works